tox.ini

```diff
@@ -37,3 +39,27 @@ commands =
 deps =
     Sphinx
     pylons-sphinx-themes
+
+
```

Comment:
Remove extra blank line

Reply:
This extra blank is for additional vertical space in between two parts in that document.

Reply:
sorry, my previous comment was about text in RELEASING.txt ; here I agree to remove the extra vspace.

Reply:
Deliberate whitespace is good. Accidental, not so much.